### PR TITLE
Fix lints so that test-all runs green

### PR DESCRIPTION
**Pull request by Empact**
With this we can set up CI on test-all for PR review, etc.

### CODE REVIEW

**Comment by Empact** [comment]
#95

File: app/components/Network/CanvasNetworkGraph.js [hunk 1]
```diff
@@ -112,7 +108,7 @@ class CanvasNetworkGraph extends Component {
 
     // remove active class
     d3.selectAll('.active-peer')
-      .each(function () {
```
Comment (JimmyMow) [inline app/components/Network/CanvasNetworkGraph.js]: es6 arrow functions lexically bind `this`, so i did not use them here on purpose. may need to disable these lines for lint

**Comment by JimmyMow** [comment]
wow this is awesome @Empact, you are killing it. Did you test the functionality locally? I might pull this branch down just to poke around and make sure things work

**Comment by Empact** [comment]
Thanks! I did test a bit, not exhaustively. I actually just synced the testnet locally so I haven't been able to run transactions or anything through it.

I took a different approach on the d3 selectAll situation - according to the docs you can call `classed` etc on the selection itself, which is more succinct/declarative and should be more performant than iterating and re-selecting.

LMK if you find anything broken, would be good to test whatever that is.

**Comment by JimmyMow** [comment]
Thanks @Empact. Been a bit busy, my apologies, I'm gonna pull this down after lunch and poke around. Will report back afterwards

**Comment by Empact** [comment]
Green on jenkins as well FYI https://travis-ci.org/Empact/zap-desktop/builds/320447675

**Comment by JimmyMow** [comment]
niiiiice @Empact! cloned it down and it's working for me locally. merging 🎉

**Pull request merged**